doc/poolset/poolset.5.md

```diff
@@ -226,6 +217,44 @@ The files in the object pool set may be created by running the following command
 $ pmempool create --layout="mylayout" obj myobjpool.set
 ```
 
+_WINUX(,
+=q=Remote replica cannot have replicas, i.e. a remote pool set file cannot
+define any replicas.=e=)
+
+
+# POOL SET OPTIONS #
```

Comment:
You should add # POOL SET OPTIONS # to TOC above, like that:
`[POOL SET OPTIONS](#notes)<br />`